### ISSUE TITLE
Wrong id selector

### ISSUE THREAD

**Issue by infusion** [WenheLI/p5.gif#13]
Hello,

in the capture constructor, you call the following:

```
this.settings.canvas = document.getElementById(`#${this.settings.canvas}`) || window.canvas;
```

getElementById does not allow to select with leading "#" - this is querySelector syntax. This means it always uses the window.canvas fallback.

Robert

**Comment by WenheLI** [comment]
You are right! I will fix it asap!

**Comment by WenheLI** [comment]
Fix by 27da75cc176fc667b07490b8944691b61259c566
